File: QubitServer/src/main/java/org/labrad/qubits/mem/StopTimerCommand.java
package org.labrad.qubits.mem;

import org.labrad.qubits.FpgaModelDac;

public class StopTimerCommand implements MemoryCommand {
  private StopTimerCommand() {}

  private static final StopTimerCommand INSTANCE = new StopTimerCommand();

  public static StopTimerCommand getInstance() {
    return INSTANCE;
  }

  public long[] getBits() {
    return new long[] {0x400001};
  }
  public double getTime_us(FpgaModelDac dac) {
    return FpgaModelDac.clocksToMicroseconds(1);
  }
}

File: QubitServer/src/main/scala/org/labrad/qubits/mem/StopTimerCommand.scala
package org.labrad.qubits.mem

import org.labrad.qubits.FpgaModelDac

object StopTimerCommand extends MemoryCommand {
  def getBits(): Array[Long] = {
    Array[Long](0x400001)
  }

  def getTime_us(dac: FpgaModelDac): Double = {
    FpgaModelDac.clocksToMicroseconds(1)
  }
}
